diff.py

```diff
@@ -1959,9 +1995,16 @@ def process(dump: str, config: Config) -> List[Line]:
         else:
             # powerpc-eabi-objdump doesn't use tabs
             row_parts = [part.lstrip() for part in row.split(" ", 1)]
+
         mnemonic = row_parts[0].strip()
         args = row_parts[1] if len(row_parts) >= 2 else ""
-        row = mnemonic + "\t" + args.replace("\t", "  ")
+
+        if i < len(lines):
+            next_row = lines[i]
+        else:
+            next_row = None
+
+        row = processor.pre_process(mnemonic, args, next_row)
```

Comment:
mnemonic and args need to be updated too, while `row` can be recomputed from those two. So make `pre_process` return Tuple[str, str]?

Reply:
looks like there is another var called `row_parts`  and now that does not get updates.. but looks like its only used for things downstream that wouldn't be affected... little confusing though

Reply:
oh, good catch. Those usages of `row_parts[1]` should be changed to `args`. And `args` should probably be strip()ped at the place where it's defined so those strip() usages can go away too

Reply:
okay I did this